### PR TITLE
Changed Max-width, height to logo & Unify gap into margin

### PR DESCRIPTION
**Pull request by yyorkchan**

### CODE REVIEW

File: resources/authgear/templates/en/web/authflowv2/setup_totp_verify.html [hunk 1]
```diff
@@ -9,11 +9,11 @@
   {{ end }}
 
   <div class="flex flex-col gap-y-8 flex-1-0-auto">
-    <div class="flex flex-col gap-y-4">
+    <div class="flex flex-col mt-2">
```
Comment (cychiuae) [inline resources/authgear/templates/en/web/authflowv2/setup_totp_verify.html:12]: We don't need margin top here coz there is no logo above.

File: resources/authgear/templates/en/web/authflowv2/wechat.html [hunk 2]
```diff
@@ -10,11 +10,11 @@
 {{ if $.WechatRedirectURI }}
 
 <div class="screen-action-layout flex-1-0-auto">
-  <header class="flex flex-col gap-4">
+  <header class="flex flex-col mt-2">
```
Comment (cychiuae) [inline resources/authgear/templates/en/web/authflowv2/wechat.html:13]: We dont need margin here

File: resources/authgear/templates/en/web/authflowv2/wechat.html [hunk 3]
```diff
@@ -62,11 +62,11 @@ <h2 class="screen-description">
 {{ else }}
 
 <div class="screen-action-layout flex-1-0-auto">
-  <header class="flex flex-col gap-4">
+  <header class="flex flex-col mt-2">
```
Comment (cychiuae) [inline resources/authgear/templates/en/web/authflowv2/wechat.html:65]: We dont need margin here

File: resources/authgear/templates/en/web/authflowv2/forgot_password.html [hunk 4]
```diff
@@ -16,9 +16,9 @@
 
   {{ if $.RequiresLoginIDInput }}
     <div class="flex flex-col gap-8 flex-1-0-auto">
-      <header class="flex flex-col gap-4">
+      <header class="flex flex-col mt-2">
```
Comment (cychiuae) [inline resources/authgear/templates/en/web/authflowv2/forgot_password.html:19]: We don't need margin top here

**Review by louischan-oursky — COMMENTED** [review]
Per the offline discussion between York and me, we will

1. Add `screen-title-description-layout.css`. It has a class `screen-title-description-layout` == `flex flex-col gap-4`.
2. Add `screen-logo-layout.css`. It has a class `screen-logo-layout` == `flex flex-col gap-8`
3. Add `screen-title-description-layout` and `screen-logo-layout` to all suitable places.

**Pull request closed without merging**